api/server/server.go

```diff
@@ -1058,14 +1058,11 @@ func (s *Server) postContainersWait(version version.Version, w http.ResponseWrit
 		return fmt.Errorf("Missing parameter")
 	}
 
-	name := vars["name"]
-	cont, err := s.daemon.Get(name)
-	if err != nil {
+	status, err := s.daemon.ContainerWait(vars["name"], -1*time.Second)
+	if err != nil && !strings.Contains(strings.ToLower(err.Error()), "timed out") {
```

Comment:
I'm ok with what you said, we are ignoring the error, I don't know if there was a reason, maybe @LK4D4 ?
this is old code:
```
status, _ := cont.WaitStop(-1 * time.Second)
```